src/tests/test_webworker.py

```diff
@@ -12,8 +16,10 @@ def test_runwebworker_different_package_name(selenium_webworker_standalone):
     assert isinstance(output, str)
 
 
-def test_runwebworker_no_imports(selenium_webworker_standalone):
+def test_runwebworker_no_imports(selenium_webworker_standalone, script_type):
     selenium = selenium_webworker_standalone
+    if selenium.browser == "firefox" and script_type == "module":
```

Comment:
Since you do this on every single test that uses `selenium_webworker_standalone`, wouldn't it be better to move this into `selenium_webworker_standalone`?